src/co_tools/co_fastq.py

```diff
@@ -0,0 +1,156 @@
+import os
+import re
+import subprocess
+import sys
+from glob import glob
+from pathlib import Path
+
+# local imports
+from .get_logger import LOGGER
+
+
+def get_fastq_pair(dir_path: str = "../data"):
+    total_dirs = 0
+    for base, dirs, files in os.walk(dir_path):
```

Comment:
Can't the input directories be input parameters instead of trying to guess at them?

Reply:
No, I designed this function this way to avoid having users enter command line arguments in the pipelines UI. Users only need to make sure 1 folder has all the files (global or collect depending on version of CO) and another folder with just 1 file.

Reply:
Right, that makes sense as a default, but couldn't there be cases where you have directories you don't want to walk? I've run into issues before where you have some tools generate crazy nested directories that take forever to walk through and it'd be nice to be able to specify up front where your global directory and non-global directory is.